src/views/vmnetworks/form/components/ProjectMapping.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { Form, Radio, Title } from '@patternfly/react-core';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+
+import { VMNetworkForm } from '../constants';
+
+import ProjectList from './ProjectList';
+import ProjectNamespaceSelector from './ProjectNamespaceSelector';
+
+const ProjectMapping: FC = () => {
+  const { t } = useNetworkingTranslation();
+  const { control, setValue, watch } = useFormContext<VMNetworkForm>();
+
+  const projectList = watch('projectList');
+
+  return (
+    <Form>
+      <Title headingLevel="h2">{t('Project Mapping')}</Title>
+      <p>
+        {'You can select projects from the list or select label to specify qualifying projects.'}
```

Comment:
This text needs to be wrapped in `t()`.

Correction: 'You can select projects from the list or select labels to specify qualifying projects.'